src/rockstor/system/osi.py

```diff
@@ -96,14 +96,14 @@ def run_command(cmd, shell=False, stdout=subprocess.PIPE,
         out = out.split('\n')
         err = err.split('\n')
         rc = p.returncode
-    except Exception, e:
+    except Exception as e:
         msg = ('Exception while running command(%s): %s' % (cmd, e.__str__()))
         raise Exception(msg)
 
     if (rc != 0):
         if (log):
-            e_msg = ('non-zero code(%d) returned by command: %s. output: %s error:'
-                     ' %s' % (rc, cmd, out, err))
+            e_msg = ('non-zero code({0}) returned by command: {1}. output: '
+                     '{2} error: {3}'.format(rc, cmd, out, err))
```

Comment:
@schakrava Is this an intended edit, ie from %d %s etc to {0}  {1}, don't we loose formatting capability this way?
Due to merge conflicts I'm hand converting my pr so need to know what you intended here.